aya/src/programs/lirc_mode2.rs

```diff
@@ -68,7 +68,8 @@ impl LircMode2 {
         let prog_fd = prog_fd.try_clone()?;
         let lircdev_fd = lircdev.as_fd().try_clone_to_owned()?;
 
-        bpf_prog_attach(prog_fd.as_fd(), lircdev_fd.as_fd(), BPF_LIRC_MODE2)?;
+        const NO_FLAGS : u32 = 0;
+        bpf_prog_attach(prog_fd.as_fd(), lircdev_fd.as_fd(), BPF_LIRC_MODE2, NO_FLAGS)?;
```

Comment:
Per comment below this would become `CgroupAttachFlags::default()`

Reply:
I've updated the fn signature to allow flags to be passed in, and updated doc tests to pass in `CgroupAttachFlags::empty()`.  Let me know if that's what you had in mind.